internal/dashboard/handler/info_handler.go

```diff
@@ -134,11 +163,32 @@ func GetStatistics(c *gin.Context) (*response.StatisticDataResponse, error) {
 	if err != nil {
 		return nil, httpErr.NewInternal(err.Error())
 	}
+
+	// Get deployment named oceanbase-controller-manager in oceanbase-system namespace
+	deployment, err := clt.ClientSet.AppsV1().Deployments("oceanbase-system").Get(c, "oceanbase-controller-manager", metav1.GetOptions{})
```

Comment:
we need to search all namespaces

Reply:
Maybe we can use `List` method and take the first one in returned items. How do you think about it?